### ISSUE TITLE
[2?] Gather free text present at nuget.org project description page and collect keywords present

### ISSUE THREAD

**Issue by fridex** [openshiftio/openshift.io#808]
# Description

As README files are not strictly present on GitHub for each project (moreover we do not have GitHub URL for each project as it is optional), it would be good to gather as much as possible free text sources and do keywords extraction on them using existing data mining algorithm implemented in tagger.

One of candidates for nuget ecosystem is nuget's.org project description page - see [NUnit as an example](https://www.nuget.org/packages/NUnit/). We should aggregate only text without any HTML markup so we change code only once if the page changes over time and we do not maintain different text "parsers".

# Acceptance criteria
 - [ ] Selinon `RepositoryDescCollectorTask` task that gathers PyPI and NPM descriptions in package level flows is modified to gather description as plain text
 - [ ] already existing `keywords_tagging` task is adjusted to extract keywords also from gathered NPM description and this result is placed on S3 (as now) - [task execution condition](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/dispatcher/flows/bayesianPackageAnalysisFlow.yml#L23) needs to be modified
 - [ ] tests present in test_repository_description.py are modified so they cover nuget ecosystem as well
 - [ ] code runs in production and is fully functional

**Comment by fridex** [comment]
Similar to https://github.com/openshiftio/openshift.io/issues/728 and https://github.com/openshiftio/openshift.io/issues/727

**Comment by miteshvp** [comment]
No longer relevant
